funcions/2.js

```diff
@@ -0,0 +1,17 @@
+function oldSum(list) {
+	var finalValue = 0
+	for (var i = 0; i < list.length; i++) {
+		finalValue += list[i]
+	}
+	return finalValue
+}
+
+function newSum(list) {
+	const finalVal = list.reduce((acc, cur) => acc + cur, 0)
+	return finalVal
```

Comment:
Nitpick: just a note for future as `eslint` would complain about having a redundant variable in here. 😆 
```suggestion
	return list.reduce((acc, cur) => acc + cur, 0)
```